src/BaselineOfSpec/MCVersionLoader.extension.st

```diff
@@ -0,0 +1,6 @@
+Extension { #name : #MCVersionLoader }
+
+{ #category : #'*BaselineOfSpec' }
```

Comment:
This should be removed for the integration in Pharo. It's a hack to update Spec.

Reply:
Done + [pharo-spec/Spec#337](https://github.com/pharo-spec/Spec/issues/337)